ballerina/Dependencies.toml

```diff
@@ -0,0 +1,45 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
+
+# This file is auto-generated by Ballerina for managing dependency versions.
+# It should not be modified by hand.
+
+[ballerina]
+dependencies-toml-version = "2"
+
+[[package]]
```

Comment:
Seems this is not updated. Re-running `./gradlew clean build` would automatically update this.